deny.toml

```diff
@@ -94,8 +44,6 @@ ignore = [
 # More documentation for the licenses section can be found here:
 # https://embarkstudios.github.io/cargo-deny/checks/licenses/cfg.html
 [licenses]
-# The lint level for crates which do not have a detectable license
-unlicensed = "deny"
```

Comment:
All these license-related configs are deprecated because now all licenses are denied unless you allow them explicitly.